tango.py

```diff
@@ -71,30 +67,29 @@ def __init__(self):
             from vmms.distDocker import DistDocker
             vmms = DistDocker()
 
-        self.vmms = {Config.VMMS_NAME: vmms}
-        self.preallocator = Preallocator(self.vmms)
+        self.preallocator = Preallocator({Config.VMMS_NAME: vmms})
```

Comment:
Why can't TangoServer just have an attribute `vmms` like the following: `self.vmms = self.preallocator.vmms`?